### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by SadiqZak**
Added reducer functionality to the chips, completed single video details page

### CODE REVIEW

**Comment by SadiqZak** [comment]
homepage

File: src/Backend/controllers/filter.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const filter = (arr, type) =>{
+    return type==="Games"
```
Comment (Devansu-Yadav) [inline src/Backend/controllers/filter.js:2]: Using `switch` statement here would be better in my opinion.

File: src/Components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.header-right{
+    order: 1;
+}
+
+
+@media screen and (max-width: 600px) {
+    .search{
+        order: 1;
```
Comment (Devansu-Yadav) [inline src/Components/Header/Header.css:64]: This is a really nice way to order flex elements using CSS.

**Review by Devansu-Yadav — APPROVED** [review]
Looks good to merge.

Like: 
- Using the `order` property to order flex elements in CSS.
- UI looks good with good color combination
- Clean and modular code.

Suggestions:
- Home page can be made more responsive on Mobile screens (especially the placement of the sidebar at the bottom of the screen is blocking the view).
- Would suggest styling the sidebar on Mobile as a menu at the bottom of screen with icon tabs to navigate to different pages.

**Pull request merged**